### PR TITLE
Fix crypto test

### PR DESCRIPTION
**Pull request by ndthanhdev**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
-    }
-}
```
Comment (vunguyenhung) [inline .eslintrc]: Why removing those rules ?

File: .eslintrc.json [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-    "es6": true,
     "node": true,
     "jest": true
   },
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
+    "ecmaVersion": 8
```
Comment (vunguyenhung) [inline .eslintrc.json]: Why we need this?

File: package.json [hunk 3]
```diff
@@ -20,8 +20,9 @@
     "debug": "^3.1.0",
     "folktale": "^2.1.0",
     "inquirer": "^5.2.0",
+    "parse-torrent": "^6.0.1",
```
Comment (vunguyenhung) [inline package.json]: Install this only when we actually need it.

File: .eslintrc.json (hunk 2 above)
Comment (ndthanhdev) [reply .eslintrc.json]: for async/await

File: package.json [hunk 4]
```diff
@@ -20,7 +20,6 @@
     "debug": "^3.1.0",
     "folktale": "^2.1.0",
     "inquirer": "^5.2.0",
-    "parse-torrent": "^6.0.1",
```
Comment (vunguyenhung) [inline package.json]: Update package-lock.json

File: .eslintrc (hunk 1 above)
Comment (ndthanhdev) [reply .eslintrc]: .eslintrc.json and .eslintrc  we should keep only one.

Comment (vunguyenhung) [reply .eslintrc]: Make sure you move ALL the config of the removed file to the remaining file and make sure the config works.

File: .eslintrc.json (hunk 2 above)
Comment (vunguyenhung) [reply .eslintrc.json]: Don’t rely on async await too much.

Comment (ndthanhdev) [reply .eslintrc.json]: It's good for simple concurrent tasks.

File: .eslintrc (hunk 1 above)
Comment (ndthanhdev) [reply .eslintrc]: rules of jest are migrated and refactored. I think experimentalObjectRestSpread is included in ecma8 .

**Pull request merged**